src/engine/enginebuffer.h

```diff
@@ -148,6 +152,10 @@ class EngineBuffer : public EngineObject {
             return tr("Soundtouch (faster)");
         case RUBBERBAND:
             return tr("Rubberband (better)");
+#if (RUBBERBAND_API_MINOR_VERSION >= 2 && RUBBERBAND_API_MINOR_VERSION >= 7)
+        case RUBBERBAND_R3:
+            return tr("Rubberband R3 (even better)");
```

Comment:
```suggestion
            return tr("Rubberband R3 (near-hi-fi quality)");
```
taken from https://breakfastquay.com/rubberband/why.html